### ISSUE TITLE
[Content/IA] Evaluate new My VA title

### ISSUE THREAD

**Issue by tressaellen** [department-of-veterans-affairs/va.gov-team#13738]
## Background

In this [Slack conversation](https://dsva.slack.com/archives/C909ZG2BB/p1600386040001800), there were some nomenclature/content inconsistencies pointed out that are worth resolving.

- My VA and My Health use "My", while the profile uses "Your". We should be consistent about how we are using these pronouns. Currently, it sounds like we should be using "Your," so we need to discuss whether we want to make those changes with My VA and My Health.
- In the dropdown navigation, there is currently "My VA", "My Health", and "Profile". There has been discussion about adding "My/Your" to profile to be consistent.

## Tasks
- [x] Discuss possible titles
- [x] Decide on a new title with the team/content

**Comment by Samara-Strauss** [comment]
I don't think there will be a title for the the logged in homepage since it's just going to be a homepage now, but we can discuss that.

There's no reason to wait on unifying the other titles in the meantime.

**Comment by Samara-Strauss** [comment]
I changed the scope of this ticket:

- We are keeping My VA and no longer retiring it, so the initial issue of "what do we call the new logged-in homepage?" is no longer relevant.
- However, in the Slack conversation, there were issues around our use of "My" vs. "Your", specifically that everything in the profile uses "Your" but My VA and My Health use "My". This is a larger issue we need to reconcile.

**Comment by callen2563** [comment]
@DanielleThierryUSDSVA @peggygannon @tressaellen 
Hi Danielle and Peggy, I just wanted to follow up on this ticket. Do you have any ideas on which direction we should go? See Samara's comment: "However, in the Slack conversation, there were issues around our use of "My" vs. "Your", specifically that everything in the profile uses "Your" but My VA and My Health use "My". This is a larger issue we need to reconcile."

**Comment by peggygannon** [comment]
Hi, @callen2563 I don't think I was on that Slack thread, so can you drop in what the issues were about My vs. Your?

**Comment by Samara-Strauss** [comment]
@peggygannon it is [this slack conversation](https://dsva.slack.com/archives/C909ZG2BB/p1600386040001800).

**Comment by peggygannon** [comment]
Hi, @Samara-Strauss @callen2563 @tressaellen oh, sorry! I didn't see that linked above. (ha, and i was on the slack from last fall.)

@mnorthuis and I discussed with the sitewide content/IA team yesterday and here are our recommendations. Please let us know if you have any questions. 
 
*Re:  in My VA and My Health use "My", while the profile uses "Your". We should be consistent about how we are using these pronouns. Currently, it sounds like we should be using "Your," so we need to discuss whether we want to make those changes with My VA and My Health.*

We looked at this and recommend removing "Your" from the Profile breadcrumb and the left nav, So these both would now read "Profile" not "Your profile" 

*Re: In the dropdown navigation, there is currently "My VA", "My Health", and "Profile". There has been discussion about adding "My/Your" to profile to be consistent.*

Team looked at this together yesterday and agreed that we are fine with have some inconsistency here. We are OK with the dropdown reading  "My VA" "My Health, and "Profile"

**Comment by Samara-Strauss** [comment]
Great! So no changes to My VA, but we will remove "Your" from "Your Profile."

@mshea0606 can you please create a ticket that documents we are removing "Your" from "Your Profile." Please document all the places in which this needs to be updated and assign to FE to make those updates accordingly.

**Comment by mshea0606** [comment]
[FE] Profile: Remove "Your" from "Your Profile" #19890, @Samara-Strauss
